### ISSUE TITLE
Tutorial or example on embarrassingly parallel/consensus MCMC

### ISSUE THREAD

**Issue by fehiepsi** [pyro-ppl/numpyro#417]
Despite that NumPyro is very fast (comparing to other frameworks), running MCMC for large datasets is still slow. Even that GPU helps us increase 10x the speed, it is still slow. This is when subposterior methods show their advantages. It is also a good chance to illustrate some available utilities [consensus](https://numpyro.readthedocs.io/en/stable/mcmc.html#numpyro.hmc_util.consensus) and [parametric](https://numpyro.readthedocs.io/en/stable/mcmc.html#numpyro.hmc_util.parametric), which merge subposteriors together, to users.

**Comment by EdwardRaff** [comment]
I have some upcoming work that I _think_ this might solve, but the documentation is very sparse. Are the `subposteriors` the states from previous NUTS runs? Can it work with the DiscreteHMCGibbs sampler?

**Comment by fehiepsi** [comment]
Yes, subposteriors come from NUTS runs with a shard of data. The method does not work with discrete ones I think.
